lib/subscribex/publisher/pool.ex

```diff
@@ -23,7 +23,8 @@ defmodule Subscribex.Publisher.Pool do
     result =
       broker
       |> group_name
-      |> :pg2.get_closest_pid()
+      |> :pg.get_members()
```

Comment:
Let's change this from `get_members` to `get_local_members` so we're not shuffling across the network unnecessarily.